airflow/bin/cli.py

```diff
@@ -424,6 +424,50 @@ def clear(args):
         confirm_prompt=not args.no_confirm,
         include_subdags=not args.exclude_subdags)
 
+def mark_success(args):
+    dag = get_dag(args)
+    dagruns = dag.get_dagruns(args.start_date, args.end_date)
+    # todo: run.dag is transient, needs to be set
+    for run in dagruns:
+        run.dag = dag
+
+    tis = []
+    for run in dagruns:
+        tis += run.mark_success(
```

Comment:
It's kind of annoying to have to do this twice.. once with dry_run and once with out.. Just complaining.